code/modules/hydroponics/hydroitemdefines.dm

```diff
@@ -512,18 +512,20 @@
 	inhand_icon_state = "scythe0"
 	lefthand_file = 'icons/mob/inhands/weapons/polearms_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/polearms_righthand.dmi'
-	force = 13
+	force = 17
```

Comment:
This is absolutely more than necessary, but if you can find someone to make some sprites, I think this would be ideal as a two-handed weapon for this force. You _almost_ gave it spear force, so I don't see why not make it similar to a spear if it had two-handed.

Reply:
13 force and no wound bonus is completely sufficient for an item which is available from a vending machine and orderable in bulk from cargo.
This item should be left as it is.

Reply:
> Additionally on the scythe, the crew really needs more good ghetto weaponry that isn't the boring same ol' of baseball bats, spears, cleavers... and making scythes useful is a great way to help bridge that gap. They deal a satisfying amount of damage now, with the clear downside, of course, being that they're bulky and hard to lug around.

You're worrying about nothing, the alternatives are still vastly superior. Scythes aren't 'orderable in bulk', especially when 'one' weed control crate costs 100 credits more than a stun baton crate, which holds three stun batons... which are vastly better than any actual weapon. The crew is often left completely in the lurch when needing to fight against non-human enemies such as spiders because all the great weapons exist only in security's hands.

It's slightly better than a surgical saw, sure, but again, it's bulky. It's also only two more damage than a welding tool.

If you're _still_ unconvinced, I'll tone the damage down, but the crew's Weapon Economy will suffer for it, for no real gain.

Reply:
The fact that there are better items around doesn't justify buffing this one or we'd be increasing the force of every item.
The crew's weapon economy isn't going to suffer, because "not buffing" an item can't possibly be a net negative.

Reply:
> This is absolutely more than necessary, but if you can find someone to make some sprites, I think this would be ideal as a two-handed weapon for this force. 
oooh, i like doing sprites. if this is still a thing i might be able to cobble one together

Reply:
> The fact that there are better items around doesn't justify buffing this one or we'd be increasing the force of every item. 

That's why I suggested making it more like a spear. It'd otherwise be comparable to something literally anyone can make from scrap.